src/getNonce.ts

```diff
@@ -0,0 +1,7 @@
+import crypto from "crypto"
```

Comment:
I think this file isn't used anymore..

Reply:
I'm wrong, maybe I saw it removed in a different version of the code during our zoom meeting.